encryption-at-rest.md

```diff
@@ -128,3 +128,11 @@ When restoring the backup, both `--s3.sse` and `--s3.sse-kms-key-id` should NOT
 ```
 ./br restore full --pd <pd-address> --storage "s3://<bucket>/<prefix> --s3.region <region>"
 ```
+
+## Encryption at rest for TiFlash <span class="version-mark">New in v4.0.5</span>
+
+TiFlash supports encryption at rest starting from v4.0.5. All files written by TiFlash (including TiFlash Proxy), including data files, schema files, temporary files, etc., are encrypted by the current data key.
```

Comment:
The senetence "All files writtern by ..." seems duplicate with the next paragraph.

Reply:
And the first two paragraph should merge to one paragraph as in its Chinese version.

Reply:
thx~updated. PTAL again.@lidezhu